webapi/src/main/scala/org/knora/webapi/util/standoff/StandoffTagUtilV2.scala

```diff
@@ -750,7 +750,8 @@ object StandoffTagUtilV2 {
                                 case Some(SmartIriLiteralV2(SmartIri(OntologyConstants.Xsd.Boolean))) =>
                                     StandoffTagBooleanAttributeV2(standoffPropertyIri = propIri, value = value.toBoolean)
 
-                                case Some(SmartIriLiteralV2(SmartIri(OntologyConstants.Xsd.Uri))) => StandoffTagIriAttributeV2(standoffPropertyIri = propIri, value = value)
+                                case Some(SmartIriLiteralV2(SmartIri(OntologyConstants.Xsd.Uri))) =>
+                                    StandoffTagUriAttributeV2(standoffPropertyIri = propIri, value = value)
```

Comment:
The wrong attribute object was constructed here.

Reply:
again: I took pain killers when I wrote this code (dental surgery)